### ISSUE TITLE
Python breakpoint insertion error

### ISSUE THREAD

**Issue by ostrbor** [syl20bnr/spacemacs#8251]
#### Description :octocat:
"spacemacs/python-toggle-breakpoint" doesn't work

#### Reproduction guide :beetle:

- Start Emacs
- open python file
- run "spacemacs/python-toggle-breakpoint"

*Observed behaviour:* :eyes: :broken_heart:

_progn: Symbol’s function definition is void: insert-string_

> I couldn't find definition of (insert-string ).
> I searched with (describe-function ) and in spacemacs github repository.

*Expected behaviour:* :heart: :smile:

insert line with "import pdb; pdb.set_trace()"

#### System Info :computer:
- OS: gnu/linux
- Emacs: 26.0.50.2
- Spacemacs: 0.200.7
- Spacemacs branch: master (rev. c2774bc)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(ansible helm auto-completion better-defaults emacs-lisp git github markdown org
         (shell :variables shell-default-term-shell "/usr/bin/zsh" shell-default-height 30 shell-default-position 'bottom)
         spell-checking syntax-checking version-control
         (python :variables python-enable-yapf-format-on-save t python-sort-imports-on-save t)
         django html yaml go docker gnus search-engine nginx imenu-list floobits sql)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND GPM DBUS GCONF GSETTINGS NOTIFY LIBSELINUX GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 MODULES XWIDGETS LIBSYSTEMD

**Comment by ostrbor** [comment]
I've just found function (insert ) in elisp and used it as alias to (insert-string ).  
If you have same problem as in this issue, this code in (user-config ) might help:
 
```
(defun insert-string (s)
    (insert s))
```

**Comment by Miciah** [comment]
`insert-string` was removed in Emacs 26.1, and Spacemacs's use of `insert-string` was changed to `insert` in the develop branch with #7676 / https://github.com/syl20bnr/spacemacs/commit/12284e16b0aa3a73edc4a55e78ff5ed62c726666.  This issue can be labeled as "Fixed in develop" and "Emacs 26.1".
